common/common.scss

```diff
@@ -181,6 +181,18 @@ label.btn i {
   }
 }
 
+// Edge case for like button (double button)
+
+.double-button > .btn-text {
```

Comment:
This has to be .btn-text specifically (targeting `.like-count` meant that it would incorrectly remove the border radius for the like count on your own posts)